test/clj/game_test/cards/events.clj

```diff
@@ -1671,20 +1671,31 @@
 
 (deftest emergent-creativity
   ;; Emergent Creativty - Double, discard programs/hardware from grip, install from heap
-  (do-game
-    (new-game {:runner {:deck ["Emergent Creativity" "Paperclip"
-                               "Heartbeat" "Gordian Blade" "Test Run"]}})
-    (starting-hand state :runner ["Emergent Creativity" "Heartbeat" "Gordian Blade" "Test Run"])
-    (take-credits state :corp)
-    (play-from-hand state :runner "Emergent Creativity")
-    (click-card state :runner (find-card "Heartbeat" (:hand (get-runner))))
-    (click-card state :runner (find-card "Gordian Blade" (:hand (get-runner))))
-    (click-prompt state :runner "Done")
-    (click-prompt state :runner (find-card "Paperclip" (:deck (get-runner))))
-    (is (= 3 (:credit (get-runner))) "Offset cost of installing Paperclip")
-    (is (zero? (count (:deck (get-runner)))) "Installed from heap")
-    (is (= 3 (count (:discard (get-runner)))) "Discard is 3 cards - EC, Heartbeat, GB")
-    (is (= 2 (:click (get-runner))) "Emergent Creativity is a Double event")))
+  (testing "Basic test"
+    (do-game
+      (new-game {:runner {:deck ["Emergent Creativity" "Paperclip"
+                                "Heartbeat" "Gordian Blade" "Test Run"]}})
+      (starting-hand state :runner ["Emergent Creativity" "Heartbeat" "Gordian Blade" "Test Run"])
+      (take-credits state :corp)
+      (play-from-hand state :runner "Emergent Creativity")
+      (click-card state :runner (find-card "Heartbeat" (:hand (get-runner))))
+      (click-card state :runner (find-card "Gordian Blade" (:hand (get-runner))))
+      (click-prompt state :runner "Done")
+      (click-prompt state :runner (find-card "Paperclip" (:deck (get-runner))))
+      (is (= 3 (:credit (get-runner))) "Offset cost of installing Paperclip")
+      (is (zero? (count (:deck (get-runner)))) "Installed from heap")
+      (is (= 3 (count (:discard (get-runner)))) "Discard is 3 cards - EC, Heartbeat, GB")
+      (is (= 2 (:click (get-runner))) "Emergent Creativity is a Double event")))
+  (testing "Resource not shown in prompt"
+    (do-game
+      (new-game {:runner {:deck ["Emergent Creativity" "Heartbeat" "Gordian Blade" "Liberated Account"]}})
+      (starting-hand state :runner ["Emergent Creativity" "Heartbeat" "Gordian Blade" ])
+      (take-credits state :corp)
+      (play-from-hand state :runner "Emergent Creativity")
+      (click-card state :runner (find-card "Heartbeat" (:hand (get-runner))))
+      (click-card state :runner (find-card "Gordian Blade" (:hand (get-runner))))
+      (is (= 1 (-> (get-runner) :prompt first :choices count)) "Liberated Account shouldn't be shown in prompt (only Cancel)")
```

Comment:
Can you switch this to something like `(= ["Cancel"] (-> (get-runner) :prompt first :choices))`? That way, we have a clear understanding of what's being checked, and a change in the prompts will make this fail instead of silently passing (because we're only counting quantity).

Reply:
Sure, I will fix it